src/v3/src/components/Widget/index.tsx

```diff
@@ -76,8 +76,6 @@ export const Widget: FunctionComponent<WidgetProps> = (widgetProps) => {
   } = widgetProps;
 
   const [data, setData] = useState<FormBag['data']>({});
-  // This data is not included when submitting to auth-js
-  const [additionalData, setAdditionalData] = useState<FormBag['data']>({});
```

Comment:
@glenfannin-okta seems like authjs can do the filter work (forgot when it's added), so I removed this global context here.